### PR TITLE
[REVIEW] Python and Cython style cleanup, pre-commit hook

### PR DESCRIPTION
**Pull request by kkraus14**

### CODE REVIEW

File: python/cuspatial/cuspatial/_lib/spatial.pxd [hunk 1]
```diff
@@ -9,33 +9,44 @@ from cudf._lib.cudf cimport *
...
 
-cdef extern from "coordinate_transform.hpp" namespace "cuspatial" nogil: 
- cdef pair[gdf_column, gdf_column] lonlat_to_coord(const gdf_scalar cam_x,
-                                                   const gdf_scalar cam_y,
-                                                   const gdf_column  & in_x,
-                                                   const gdf_column  & in_y) except +
+cdef extern from "coordinate_transform.hpp" namespace "cuspatial" nogil:
+    cdef pair[gdf_column, gdf_column] lonlat_to_coord(
+        const gdf_scalar& cam_x,
+        const gdf_scalar& cam_y,
```
Comment (kkraus14) [inline python/cuspatial/cuspatial/_lib/spatial.pxd]: These weren't pass by reference before, but the `.hpp` file had them as pass by reference so I updated it. If incorrect please let me know.

File: python/cuspatial/cuspatial/tests/test_lonlat2coord.py [hunk 2]
```diff
@@ -1,98 +1,98 @@
...
     x = [-180, 180, -180, 180]
     y = [-90, 90, 90, -90]
-    result = cuspatial.lonlat_to_xy_km_coordinates(x[corner], y[corner],
-        cudf.Series(x[corner]),
-        cudf.Series(y[corner])
+    result = cuspatial.lonlat_to_xy_km_coordinates(
+        x[corner], y[corner], cudf.Series(x[corner]), cudf.Series(y[corner])
     )
-    result = cudf.DataFrame({'x': [0], 'y': [0]})
+    assert_eq(result, cudf.DataFrame({"x": [0], "y": [0]}))
```
Comment (kkraus14) [inline python/cuspatial/cuspatial/tests/test_lonlat2coord.py]: Not sure if this was what the test was meant to do.

@thomcom?

Comment (thomcom) [reply python/cuspatial/cuspatial/tests/test_lonlat2coord.py]: Nice catch, thanks.

Comment (kkraus14) [reply python/cuspatial/cuspatial/tests/test_lonlat2coord.py]: Had to make them floats, but test passes for me now, woohoo!

**Pull request merged**